lib/autofs_utils.pm

```diff
@@ -134,6 +134,11 @@ sub configure_service {
         test_conf_file_content => $test_conf_file_content,
         test_mount_dir         => $test_mount_dir);
 
+    if (script_run('test -f /etc/nsswitch.conf') && !script_run('rpm -q glibc')) {
```

Comment:
In which case would `rpm -q glibc` fail?

Reply:
Oops, I meant `glibc-devel`. Thanks!

Reply:
but can be skipped anyway